packages-exp/auth-exp/test/integration/webdriver/util/test_runner.ts

```diff
@@ -57,7 +57,8 @@ export function browserDescribe(
 setTimeout(() => {
   for (const browser of BROWSERS) {
     describe(`Testing in browser "${browser}"`, () => {
-      before(async () => {
+      before(async function () {
+        this.timeout(20000); // Starting browsers can be slow.
```

Comment:
Don't you have to type the this variable?

Reply:
Nope :) automatic